continuous_integration/kubernetes/script.sh

```diff
@@ -2,3 +2,14 @@
 
 echo "Running tests..."
 kubectl exec dask-gateway-tests -n dask-gateway /working/continuous_integration/kubernetes/script-internal.sh
+
+echo "Running chartpress"
+pushd $TRAVIS_BUILD_DIR/resources/helm/
+docker build -t dask-gateway-base images/dask-gateway-base
+chartpress --commit-range ${TRAVIS_COMMIT_RANGE} --tag `date +%y.%m.%d`
```

Comment:
Did you intend to use CalVer here? I would expect we want to use the git tag instead.

```suggestion
chartpress --commit-range ${TRAVIS_COMMIT_RANGE} --tag ${TRAVIS_TAG}
```